icons/raylib.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Raylib</title><path d="M0 24h24V0H0Zm22.5-1.5h-21v-21h21zm-6.187-2.25h.468c.44 0 .469-.019.469-.234 0-.188-.047-.235-.234-.235h-.235V16.97h-.468v3.281zm1.406-3.047c0 .188.047.235.234.235.188 0 .235-.047.235-.235 0-.187-.047-.234-.235-.234-.187 0-.234.047-.234.234zm.937 3.047H21v-2.344h-1.875v-.468c0-.441-.019-.47-.234-.47h-.235zm1.875-.469h-1.406v-1.406h1.406zm-12.656.469h.469v-1.875h1.875v-.469H7.875Zm2.813-1.875h1.874v.469h-1.874v1.406h2.343v-2.344h-2.344v.469zm1.874 1.172c0 .225-.018.234-.703.234-.684 0-.703-.01-.703-.234 0-.225.019-.235.703-.235.685 0 .704.01.704.235zm.938.703h1.875v.469H13.5v.468h2.344v-3.28h-.469v1.874h-1.406v-1.875H13.5Zm4.219 0h.468v-2.344h-.468z"/></svg>
```

Comment:
![image](https://github.com/simple-icons/simple-icons/assets/35000807/20be005f-d8ae-4a0e-8ac7-242414e36ca9)

The text inside the box is not matching. The original icon has no curves at all. The text letters have squared corners.